modules/ROOT/pages/dw-multipart-types.adoc

```diff
@@ -7,13 +7,13 @@
 | `type Multipart = { preamble?: String, parts: { _?: MultipartPart } }`
 | `MultiPart` type, a data structure for a complete `Multipart` format. See the
 output example for the Multipart `form` function
-https://docs.mulesoft.com/mule-runtime/4.3/dw-multipart-functions-form[documentation].
```

Comment:
TODO: THIS IS https://docs.mulesoft.com/mule-runtime/latest/dw-multipart-functions-form[documentation] IN THE latest source.